src/core/_color-utilities.scss

```diff
@@ -1,3 +1,7 @@
+.text-error {
```

Comment:
@proand - This might not be best practice, but necessary.

Reply:
Thats ok 👍